web/07-analysis.php

```diff
@@ -190,6 +190,9 @@ function mapsLoaded() {
       <label>Ensemble<sup>*</sup></label>
       <input type="text" name="ensemble" id="ensemble" value="<?php echo 1; ?>" onChange="validate();"/>
       <div class="spacer"></div>
+      <label>Sensitivity<sup>*</sup></label>
+      <input type="text" name="sensitivity" id="sensitivity" value="<?php echo "-1,1"; ?>" onChange="validate();"/>
```

Comment:
add title with some text, for example
"Allows to set the sensitivity analysis values as sigma, for example -1, 0, 1"